docs/upgrade/upgrade_to_v0.18.0.md

```diff
@@ -0,0 +1,97 @@
+# Upgrade to v0.18.0
+
+There are breaking changes between OpenPAI v0.14.0 and v0.18.0. Before v0.18.0, OpenPAI was based on Yarn and Kubernetes, and data was managed by HDFS. In v0.18.0, it has switched to a pure Kubernetes-based architecture. Many new features, such as `AAD authorization`, `Hivedscheduler`, `Kube Runtime`, `Marketplace`, etc., are also included in this release.
+
+The upgrade contains three phases:
+
+  1. Save previous data to a different place.
```

Comment:
Backup data

Reply:
Actually, it is not a "backup", since the data cannot be imported to the new version.